CHANGES.rst

```diff
@@ -267,6 +267,12 @@ resample
 
 - Fixed a bug in the definition of the output WCS for NIRSpec. [#7359]
 
+scripts
```

Comment:
These change log entries are buried way down in the 1.9.0 release section. They both need to be moved up to the current 1.9.5 (unreleased) section.

Reply:
Done